docs/pages/router/overview.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Expo Router
+description: Welcome to Expo Router
+sidebar_title: Overview
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+Expo Router is a routing library for React Native applications built with Expo. It provides a way to manage navigation between screens in your app, allowing users to move between different parts of your app's UI seamlessly, using the same components on multiple platforms (iOS, Android, Web, etc).
+
+Expo Router brings the best routing concepts from the web to native iOS and Android apps. Every file in the **app** directory automatically becomes a route in your mobile navigation, making it easier than ever to build, maintain, and scale your project. It's built on top of our powerful [React Navigation suite](https://reactnavigation.org/) enabling truly native navigation. The entire deep linking system is automatically generated live, so users can share links to any route in your app.
+
+## Quick Start
```

Comment:
```suggestion
## Quick start
```

Heading in doc has to be sentence case as per our writing styelguide: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#headings